packages/ra-ui-materialui/src/input/FileInput.js

```diff
@@ -75,13 +76,15 @@ export class FileInput extends Component {
     }
 
     onDrop = files => {
-        const updatedFiles = this.props.multiple
+        const multiple = this.props.options.multiple || this.props.multiple;
```

Comment:
I think we should support only one of the two. As it is a very common usecase, I wonder if it should be `this.props.multiple` in which case, we should merge it in the options before passing them to the `DropZone` component. This should also be documented if chosen.

Reply:
The code must be updated as well. We currently check for `this.props.multiple` but it should be `this.props.options.multiple`

Reply:
Nvm, we actually pass `this.props.multiple` to the DropZone...